third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -42,13 +44,21 @@ PjRtComputationClient::PjRtComputationClient() {
   std::string device_type = sys_util::GetEnvString(env::kEnvPjRtDevice, "");
   if (device_type == "CPU") {
     TF_VLOG(1) << "Initializing PjRt CPU client...";
-    client_ = xla::GetCpuClient(/*asynchronous=*/false).ValueOrDie();
+    client_ = std::move(xla::GetCpuClient(/*asynchronous=*/false).ValueOrDie());
+  } else if (device_type == "TPU") {
+    TF_VLOG(1) << "Initializing PjRt TPU client...";
+    client_ = xla::GetTpuClient(/*max_inflight_computations=*/1).ValueOrDie();
```

Comment:
Is `max_inflight_computations=1` a short term thing or long term thing? What's it for?

Reply:
Short term. I can tune a better value once I get some non-trivial models running across all 8 cores.

Reply:
As I understand it, this is the maximum number of asynchronous operations allowed at a time. For JAX this defaults to 32: [JAX_TPU_MAX_INFLIGHT_COMPUTATIONS](https://github.com/tensorflow/tensorflow/blob/44e84cebef5a89a0840f8d401819c25af41ec3db/tensorflow/compiler/xla/python/xla_client.py#L101)